test/extended/prometheus/prometheus_builds.go

```diff
@@ -48,6 +49,11 @@ var _ = g.Describe("[sig-instrumentation][sig-builds][Feature:Builds] Prometheus
 
 	g.Describe("when installed on the cluster", func() {
 		g.It("should start and expose a secured proxy and verify build metrics", func() {
+			if e2e.TestContext.Provider == ibmcloud.ProviderName {
+				e2eskipper.Skipf("Prometheus in IBM ROKS clusters does not collect metrics from the OpenShift controller manager because it lives outside of the cluster. " +
+					"The openshift_build_total metric expected by this test is reported by the OpenShift controller manager")
```

Comment:
please link to to a JIRA tracking if this should move to openshift state metrics.